app/config/config.yml

```diff
@@ -86,10 +85,12 @@ assetic:
             output: css/min.css
         common_css:
             inputs:
-                - "bundles/website/css/common.css"
+                - "@WebsiteBundle/Resources/public/css/common.css"
```

Comment:
We can use `yuicompressor-2.4.8.jar` from `min.css` distrib, but I need to change documentation and perform some changes in build, because we will need java to run it.